miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/miniapp/Lister.kt

```diff
@@ -2,4 +2,7 @@ package com.rakuten.tech.mobile.miniapp.miniapp
 
 import com.rakuten.tech.mobile.miniapp.api.ApiClient
 
-internal class Lister(client: ApiClient)
+internal class Lister(val apiClient: ApiClient) {
+
+    suspend fun fetchMiniAppList() = apiClient.list()
```

Comment:
> MiniAppInfo and ListingEntity must combine into one. I think this can be done in the same PR or in the next one when we update because of the changes in backend API specs.

Regarding this: I don't think we should combine `MiniAppInfo` and `ListingEntity` because it will expose information that the App shouldn't know about (the files list and versionId). Also, that will make our public interface dependent on the property names that are in the response from the API.

So I think we need to keep both, and then in `Lister` we can convert the `ListingEntity` items to `MiniAppInfo` items. What do you think?

Reply:
As we talked about it, we will merge them into one and control the visibility via `internal` and name conflicts via `SerializedName`.